types/pulltorefreshjs/index.d.ts

```diff
@@ -1,10 +1,15 @@
-// Type definitions for pulltorefreshjs 0.1
+// Type definitions for pulltorefreshjs 0.2
```

Comment:
This should stay at `0.1`

Reply:
Fixed. I thought that I definitely have to increase this, but yes, I agree.